scripts/run-ci-build-wheel.sh

```diff
@@ -36,14 +35,13 @@ echo "::endgroup::"
 
 echo "::group::Install build system"
 pip install ninja numpy
-pip install --upgrade setuptools==69.5.1 wheel build
+pip install --upgrade setuptools wheel build
 echo "::endgroup::"
 
 
 echo "::group::Build wheel for FlashInfer"
-cd "$PROJECT_ROOT/python"
```

Comment:
Why remove the `+cu` info?

Reply:
local_version is concatenated to package version here: https://github.com/flashinfer-ai/flashinfer/blob/b1b1fb8fb36a6442067a4d6ceb64319a40d854d4/setup.py#L55

I can produce correct wheel name locally but it doesn't work for nightly wheels.

Reply:
I see